cmd/dcrdata/public/scss/bootstrap_overrides.scss

```diff
@@ -16,3 +16,7 @@
     overflow: visible;
   }
 }
+
+.small {
+  font-size: 80%;
+}
```

Comment:
The `small` bootstrap class for texts also changed. The new version 5 one applies a `font-size: .875em;`. This overrides it to the previous setting that dcrdata used.